shared/settings/nav/index.desktop.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import React, {Component} from 'react'
+import {globalStyles, globalColors, globalMargins} from '../../styles'
+import {Box, Badge, ClickableBox, Text} from '../../common-adapters'
+
+import type {Props, BannerType, SettingsItem as SettingsItemType} from './index'
+
+function Banner ({element, type}: {element: React$Element<*>, type: BannerType}) {
+  return (
+    <Box style={{...commonBannerStyle, ...variantBannerStyle[type]}}>
```

Comment:
This is pretty similar to the banner code in https://github.com/keybase/client/blob/master/shared/common-adapters/standard-screen.desktop.js#L11. Could use `StandardScreen` here, though it maybe overkill. It also tries to center the contents of the screen, which is inappropriate in this case -- would probably need to add a flag.